src/dataset-builder/ConceptSelector.test.ts

```diff
@@ -45,7 +46,7 @@ describe('ConceptSelector', () => {
     renderSelector();
     // Act
     const user = userEvent.setup();
-    await user.click(screen.getByLabelText('add'));
+    await user.click(screen.getByLabelText(`add ${initialRows[0].id}`));
```

Comment:
can you explain this change? Now it's adding the first concept in the list - is that the same as it was doing before? Why didn't it need the row id before?

Reply:
Oh this is just because you updated the labels for each row, is that right?